tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "esModuleInterop": true,
+      "target": "es6",
+      "moduleResolution": "node",
+      "outDir": "./lib",
```

Comment:
` "outDir": "./lib",`  removed to prevent confusion since our output is in `dist`